_data/projects/EnumFP.yml

```diff
@@ -0,0 +1,14 @@
+name: EnumFP
+desc: Functional Enum type / Sum type for javascript with simple pattern matching
+site: https://github.com/phenax/enum-fp
+tags:
+- enum
+- functional
+- functional-programming
+- sum-type
+- pattern
+- pattern-matching
+- javascript
+upforgrabs:
+  name: help wanted
+  link: https://github.com/phenax/enum-fp/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

Comment:
@phenax Please change this to https://github.com/phenax/enum-fp/labels/help%20wanted

Reply:
@ritwik12, Done